markdown/chapters/appendices/resources.md

```diff
@@ -68,8 +70,21 @@ Assume English unless specified otherwise.
 Videos from non-regular streamers or Doomsday content creators.  
 Assume English unless specified otherwise.
 
+### 2022
+
+ - Awaiting Content :D
+
 ### 2021
 
+- [MTGBlackSet display some Paper Meandeck in action vs Death and Taxes (December 2021) - German/Deutsch](https://www.youtube.com/watch?v=Hn3no4hz00s)
+- [BoshNRoll runs a league with UB Meandeck ft. Death's Shadow (December 2021)](https://www.youtube.com/watch?v=sbx6_WIuSR4)
+- [MTGBlackSet display some Paper Meandeck in action vs Moon Stompy (November 2021) - German/Deutsch](https://www.youtube.com/watch?v=898Kqa1wiGw)
```

Comment:
Could go next to the other link (add the `s` to `display`). Minor nitpick, if "ft." takes a dot, should "vs" as well. I've always written it with personally

Reply:
Moved and added .